frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -52,13 +58,23 @@ const applicationAdmins = shallowRef<FamAppAdminGetResponse[]>(
 const applicationsUserAdministers = computed(() => {
     return LoginUserState.getApplicationsUserAdministers();
 });
+const delegatedAdmins = shallowRef<FamAccessControlPrivilegeGetResponse[]>(
+    props.delegatedAdmins
+);
 
 onUnmounted(() => {
     resetNotification();
 });
 
 const onApplicationSelected = async (e: DropdownChangeEvent) => {
     setSelectedApplication(e.value ? JSON.stringify(e.value) : null);
+
+    if (LoginUserState.hasAccess(APP_ADMIN_ROLE)) {
```

Comment:
this if statement should be under the else condition below. When select FAM, there is no delegated admin for fam. So this only happens when select other apps

Reply:
Hello, Catherine! I'll make the change, thanks for letting me know!